packages/atomic/vitest.config.ts

```diff
@@ -48,6 +48,7 @@ export default defineConfig({
       'src/**/search-layout.spec.ts',
     ],
     restoreMocks: true,
+    snapshotSerializers: ['vitest-utils/lit-serializer.ts'],
```

Comment:
We could not have it everywhere and register it on demand, but I think it would serve us better to have it everywhere.